vhdbuilder/packer/packer_source.sh

```diff
@@ -262,6 +265,8 @@ copyPackerFiles() {
 }
 
 cpAndMode() {
-  src=$1; dest=$2; mode=$3
+  src=$1
```

Comment:
nit - for the sake of reviews I appreciate avoiding mixing purely stylistic changes and functional changes. I'm happy to take style changes just makes reviewing real diffs harder.

Reply:
You're right of course and in my heart I agree. I'll leave this one since the damage is done but next time I'll do a separate PR.